contracts/VaultV3.vy

```diff
@@ -547,5 +550,16 @@ def processReport(strategy: address) -> (uint256, uint256):
 #     # TODO: change feemanager contract
 #     return
 
+# Role management
+@external
+def set_role(account: address, role: Roles):
+    assert msg.sender == self.governance
+    self.roles[account] = role
 
+@internal
+def _enforce_role(account: address, role: Roles):
+    assert role in self.roles[account] # dev: not allowed
 
+@external
+def transfer_governance(governance: address):
```

Comment:
do we need an accept_governance function? to avoid misallocations of governance

do we need a refuse_governance function? to be able to "burn" the keys

Reply:
should we have a role owner for each role? atm all roles are managed by governance which seems like it is against the spirit of decentralization that v3 is after. 

re: burning keys, to burn keys, you can just transfer it to the ZERO_ADDRESS, no?

Reply:
re burning keys: yes, but if we add accept_governance we can't do that

so either accept_governance and refuse_governance or none of them

Reply:
I do not like how every role has a role manager in the previous version.
If we want more granular control over role management it's better to make a contract on top that becomes governance and have some more granular controls over the roles.
At first, governance is a multisig later it should become a contract that can be governed by veYFI.